src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/pathswap/action/OnFinishedWithErrorAction.java

```diff
@@ -36,5 +36,9 @@ public OnFinishedWithErrorAction(FlowOperationsDashboardLogger dashboardLogger)
     public void execute(State from, State to, Event event, FlowPathSwapContext context, FlowPathSwapFsm stateMachine) {
         dashboardLogger.onFailedFlowUpdate(stateMachine.getFlowId(), stateMachine.getErrorReason());
         stateMachine.saveActionToHistory("Failed to swap paths for the flow", stateMachine.getErrorReason());
+
+        log.info("Flow {} path swap failed", stateMachine.getFlowId());
```

Comment:
maybe we should use warn/error log level here?

Reply:
The warn level is now used here.